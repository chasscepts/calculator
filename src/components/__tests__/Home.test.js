import React from 'react';
import renderer from 'react-test-renderer';
import ComponentWrapper from '../test_helpers/ComponentWrapper';
import Home from '../Home';

it('renders correctly', () => {
  const tree = renderer
    .create(<ComponentWrapper component={Home} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
