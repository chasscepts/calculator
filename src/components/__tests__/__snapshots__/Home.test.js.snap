// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
        "padding": "15px 30px",
      }
    }
  >
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul
        style={
          Object {
            "display": "flex",
            "listStyle": "none",
          }
        }
      >
        <li
          style={
            Object {
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          style={
            Object {
              "borderLeft": "1px solid #ddd",
              "borderRight": "1px solid #ddd",
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          style={
            Object {
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/quote-of-the-day"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  <div
    className="content"
  >
    <h2>
      Welcome to our page
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempo incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempo incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
  </div>,
]
`;
